### DISCUSSION TITLE
[Question]: How big is the default file size limit for uploading files

### DISCUSSION THREAD

**Discussion by iamNoah1** [danny-avila/LibreChat#3061] [Troubleshooting] [answered]
### What is your question?

I did not find the default value for the maximum supported file size in the documentation.

### More Details

When looking through the code, I found something but when I understand correctly it says that the default is 512 MB. Can that be true? 

However, without configuring anything regarding the filesize not even using librechat.yaml I get a `413 Request Entity Too Large` in the browser already when trying to upload a file that is bigger than 1MB

### What is the main subject of your question?

Documentation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
That's an NGINX error, you need to add this to your nginx.conf, which is already configured in the project config file:

https://github.com/danny-avila/LibreChat/blob/4416f69a9b299c03dd947e52a033af202672597d/client/nginx.conf#L13-L15

**Reply by danny-avila** [reply]
Also I now recommend usage of Traefik over nginx if you prefer:

https://www.librechat.ai/docs/remote/traefik

**Reply by iamNoah1** [reply]
Aaaah yes, I see. Thanks for the hint :)

**Reply by Daniel-DDV** [reply]
Thanks @danny-avila had the same issue and worked for me too. Love LibreChat! Keep up the great work.
